TACC/Core-Portal
# Task/FP-661 - Edit error message in push keys modal

## Overview: ##
Moved the error message from the top of the systems push keys modal to the left side of the modal's footer. Also removed the warning message from the "Authenticate" button.

## PR Status: ##


## Related Jira tickets: ##

* [FP-661](https://jira.tacc.utexas.edu/browse/FP-661)

## Summary of Changes: ##
Moved Error Message to left side of footer
Removed alert icon from submit button

## Testing Steps: ##
1. SSH into frontera and rename your `authorized_keys` file in the `.ssh/` to something like `authorized_keys.bak`
2. Open the push keys modal in the Frontera data files area
3. Verify that the changes have been made

## UI Photos:
**BEFORE**
<img width="765" alt="keys_modal_error1" src="https://user-images.githubusercontent.com/29575979/101411562-cffb1f80-38a6-11eb-8b17-11be5e4de14b.png">
**AFTER**
<img width="782" alt="keys_modal_error2" src="https://user-images.githubusercontent.com/29575979/101411566-d1c4e300-38a6-11eb-8957-a07926f81891.png">
